iam-role.tf

```
@@ -4,6 +4,8 @@ resource "aws_iam_role" "this" {
   name                 = "${var.function_name}-${local.region_name}"
   assume_role_policy   = join("", data.aws_iam_policy_document.assume_role_policy.*.json)
```

@natemccurdy it's annoying you're getting dinged on this, but we need to update... 

```suggestion
  assume_role_policy   = join("", data.aws_iam_policy_document.assume_role_policy[*].json)
```